default.nix

```diff
@@ -161,6 +161,8 @@ let
 
       molcas = self.molcas2102;
 
+      molcas1809 = callPackage ./pkgs/apps/openmolcas/v18.09.nix { };
```

Comment:
Do we really need molcas-18.09 if it also runs with a newer version?

Reply:
Hm, I guess we don't strictly need it. I would still keep it. I guess it is a good option to have the officially supported version available.

Reply:
OK, then let's keep it for now.